cypress.json

```diff
@@ -2,5 +2,9 @@
   "baseUrl": "http://localhost:4200/#",
   "email": "123@gmail.com",
   "password": "123",
+  "test_email": "123@gmail.com",
```

Comment:
what are these emails and passwords? its not clear from variable names where these are used